packages/ember-cli-eyeglass/README.md

```diff
@@ -154,6 +154,35 @@ The contents of `my-engine/addon/styles/my-addon.scss` will be added to `dist/en
 The contents of `my-engine/addon/styles/secondary.scss` will be added to `dist/engine-dist/my-engine/assets/engine.css`
 The contents of `my-engine/addon/styles/_shared.scss` will only be included if `my-addon.scss` or `secondary.scss` explicitly import them.
 
+### Embroider Support
+
+This addon works with Embroider builds as well as classic Ember CLI builds. By default, the addon will determine if your app is using Embroider by checking for the presence of `@embroider/core` in the `dependencies` or `devDependencies` listed in `package.json`.
+
+If you have added code to `ember-cli-build.js` that _conditionally_ uses Embroider to build your application, you can provide the `embroiderEnabled` configuration option to declare to `ember-cli-eyeglass` whether you are using Embroider for this build.
```

Comment:
Would addons needing to support Embroider have to do anything for this?

Reply:
The app controls the build, so I don't think addons that are depended on by a conditional Embroider app would need to do anything.